internal/agent/f5/endpoints_test.go

```diff
@@ -142,6 +142,9 @@ const PostBigIPFixture = `{
           "profileL4": {
             "bigip": "/Common/cc_fastL4_profile"
           },
+          "profileTCP": {
+            "bigip": "/Common/cc_tcp_profile"
```

Comment:
The desired profile name is `cc_tcp_archer_profile`, is this a typo or this is not referring to the new profile..?